src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -42,7 +42,7 @@ public ConsoleLogger(string name, Func<string, LogLevel, bool> filter, bool incl
             Filter = filter ?? ((category, logLevel) => true);
             IncludeScopes = includeScopes;
 
-            if (RuntimeEnvironmentHelper.IsWindows)
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
I do not know how this is compiling under dnx451....but since the type does not exist anywhere but on CoreFX, mono is failing due to not being able to load the RuntimeInformation type. Also notice that travis had failed on this PR.